src/backend/distributed/commands/trigger.c

```diff
@@ -43,6 +43,7 @@
 
 
 /* local function forward declarations */
+static char * GetEnableDisableTriggerCommand(Oid triggerId);
```

Comment:
It may be confusing because this is not actually an `ALTER TRIGGER .. STATE ..` command but also I'm not sure about `GetEnableDisableTriggerCommand` either :D

Reply:
GetAlterTriggerStateCommand looks slightly better to me since it at least emphasizes the fact that it returns an "ALTER" command that alters the "state" :)